File: doc/_i18n/jp/tutorials/introduction/com-controller/switch_com_sources.py
def switch_com_target(self):
    # self.comZeroは、質量中心タスクのリセット後、
    # reset関数内で以下のように取得される:
    # self.comZero = self.comTask.com()
        self.comTask.com(self.comZero - eigen.Vector3d(0, 0, 0.2))
    else:
        self.comTask.com(self.comZero)
    self.comDown = not self.comDown

File: doc/_i18n/jp/tutorials/introduction/com-controller/switch_com_sources.cpp
void MyFirstController::switch_com_target()
{
  // comZeroは、reset関数内で
  // 以下のように取得される:
  // comZero = comTask->com();
  if(comDown)
  {
    comTask->com(comZero - Eigen::Vector3d{0, 0, 0.2});
  }
  else
  {
    comTask->com(comZero);
  }
  comDown = !comDown;
}
